### ISSUE TITLE
Incorrect JS implementation of `encodePacked_16_8_128`

### ISSUE THREAD

**Issue by bh2smith** [gnosis/dex-contracts#68]
This is only correct in the cases when `a`, `b` and `c` are < 16. It seems that the 'hex' is not doing what we had hoped for.

https://github.com/gnosis/dex-contracts/blob/559691a3c1f8e65d3d200a126726764436e985a9/test/snapp_utils.js#L49-L55

I could really use some help with this one. 

Should be getting something like

```
(3, 3, "18000000000000000000") --> 
[0,0,0,0,0,0,0,0,0,0,0,0,0,0,3,3,0,0,0,0,0,0,0,0,249,204,216,161,197,8,0,0]`
```
But instead it is being literally translated to 

```
(3, 3, "18000000000000000000") --> 
[0,0,0,0,0,0,0,0,0,0,0,0,0,0,3,3,0,0,0,0,0,0,18,0,0,0,0,0,0,0,0,0]
```

Some proposed suggestions about mimicking this 

https://github.com/gnosis/dex-contracts/blob/559691a3c1f8e65d3d200a126726764436e985a9/contracts/SnappBase.sol#L299-L301 

solidity functionality in JS are maybe to use `Buffer.from(value, 'utf8')` instead of hex...

**Comment by fleupold** [comment]
How is this blocking the other PR? Isn’t solidity the source of truth that we care about?

**Comment by bh2smith** [comment]
@fleupold  - In order to claim withdraw, the script needs to compute the Merkle Proof of the request and the roots needs to agree.

**Comment by bh2smith** [comment]
Closed by #70
